src/content/docs/logs/ui-data/query-time-parsing.mdx

```diff
@@ -0,0 +1,419 @@
+---
+title: Query time parsing in logs
+tags:
+  - Logs
+  - Log management
+  - UI and data
+metaDescription: How New Relic uses parsing and how to send customized log data.
+freshnessValidatedDate: 2024-05-15 
+---
+
+
+
+import logsExampleLogRecordtoHighlight from 'images/logs_screenshot-crop_example-log-record-to-highlight.webp'
+
+import logsLogsTableEntryPoint from 'images/logs_screenshot-full_logs-table-entry-point.webp'
+
+import logsLogsDetailEntryPoint from 'images/logs_screenshot-full_logs-detail-entry-point.webp'
+
+import logsOpenQueryTimeParsingEditor from 'images/logs_screenshot-full_open-query-time-parsing-editor.webp'
+
+import logsExtractValuesinParserRule from 'images/logs_screenshot-crop_extract-values-in-parser-rule.webp'
+
+import logsEditorShowsParsedDataPreview from 'images/logs_screenshot-crop_editor-shows-parsed-data-preview.webp'
+
+import logsNrqlBehindQueryRule from 'images/logs_screenshot-crop_nrql-behind-query-rule.webp'
+
+import logsDeleteQueryTimeRule from 'images/logs_screenshot-crop_delete-query-time-rule.webp'
+
+import logsViewResultsinLogDetailsView from 'images/logs_screenshot-full_view-results-in-log-details-view.webp'
+
+import logsLogTableResults from 'images/logs_screenshot-full_log-table-results.webp'
+
+import logsRuleManager from 'images/logs_screenshot-crop_rule-manager.webp'
+
+import logsQueryTimeSearchBar from 'images/logs_screenshot-full_query-time-search-bar.webp'
+
+import logsAlertQueryBuilder from 'images/logs_screenshot-crop_alert-query-builder.webp'
+
+import logsQueryBehindParsingRule from 'images/logs_screenshot-crop_query-behind-parsing-rule.webp'
+
+import logsParsingAttributesView from 'images/logs_screenshot-crop_parsing-attributes-view.webp'
+
+
+
+
+Are you looking for a quick way to list log records that contain certain strings? Query time parsing lets you parse your logs directly in the UI without needing to write complex regular expressions or Grok patterns. You can use query time parsing to temporarily extract values from your logs and quickly perform a query on these variables. The results are shown instantly since parsing is performed at query time. 
```

Comment:
I like having the the call to action at the beginning! If possible, I'd like to suggest a slightly modified CTA that highlights the differences from ingest parsing. Let me know your thoughts. I'm also open to other suggestions!

```
Are you looking for a quick way to visually extract attributes from your logs after they've been ingested into New Relic?
```

Reply:
@mchavez-newrelic this is great! I'll use your sentence!